CHANGELOG.md

```diff
@@ -16,6 +16,12 @@ For more details see the specific version's [README](https://github.com/deptofde
 
 -->
 
+## 0.3.0 - 6/1/2020
```

Comment:
Update date

Reply:
Thanks, I actually did, but didn't save the file :/ .